perception/autoware_lidar_centerpoint/lib/centerpoint_trt.cpp

```diff
@@ -135,7 +154,13 @@ bool CenterPointTRT::preprocess(
   if (!is_success) {
     return false;
   }
-  const auto count = vg_ptr_->generateSweepPoints(points_d_.get(), stream_);
+
+  const std::size_t count = vg_ptr_->generateSweepPoints(points_aux_d_.get(), stream_);
+  const std::size_t random_offset = std::rand() % config_.cloud_capacity_;
+  shufflePoints_launch(
+    points_aux_d_.get(), shuffle_indices_d_.get(), points_d_.get(), count, config_.cloud_capacity_,
+    random_offset, stream_);
```

Comment:
Any specific reasons for not adding `CHECK_CUDA_ERROR`? (Sorry if I misunderstand CHECK_CUDA_ERROR)
```suggestion
  CHECK_CUDA_ERROR(shufflePoints_launch(
    points_aux_d_.get(), shuffle_indices_d_.get(), points_d_.get(), count, config_.cloud_capacity_,
    random_offset, stream_));
```